security/pkg/nodeagent/sds/secretmanager.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sds
+
+import "time"
+
+// SecretManager defines secrets management interface which is used by SDS.
+type SecretManager interface {
+	GetSecret(proxyID, token string) (*SecretItem, error)
+}
+
+// SecretItem is the cached item in in-memory secret store.
+type SecretItem struct {
+	CertificateChain []byte
+	PrivateKey       []byte
+	Token            string
```

Comment:
Particuarlly, this token worth document what it is and how it's used.

Reply:
added comments.